go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/transparency-dev/formats
 go 1.21
 
 require (
+	github.com/google/certificate-transparency-go v1.1.8
```

Comment:
As discussed, please open an issue about this dependency. Copying out the CT bits we need is reasonable as a solution. We worked very hard to keep the number of deps for this repo to a minimum. This now implicitly depends on trillian, which is a big dependency load that was a prime part of the motivation towards creating these smaller repos.

Reply:
https://github.com/transparency-dev/formats/issues/105